prow/gerrit/gerrit.go

```diff
@@ -81,11 +81,10 @@ func NewController(instance, storage string, projects []string, kc *kube.Client,
 	if storage != "" {
 		buf, err := ioutil.ReadFile(storage)
 		if err == nil {
-			unix, err := strconv.ParseInt(string(buf), 10, 64)
-			if err != nil {
-				return nil, err
-			} else {
+			if unix, err := strconv.ParseInt(string(buf), 10, 64); err == nil {
```

Comment:
can be changed to:
```
if unix, err := strconv.ParseInt(string(buf), 10, 64); err != nil {
	return nil, err
}
lastUpdate = time.Unix(unix, 0)
```
saves one line.
the `} else {` in the original was not needed too.

Reply:
variable unix is only inside the if scope , if I were to try do what you suggest above would not work as I cannot access variable ``unix``

Reply:
i guess, that's true!
in that case the original is fine.